shared/src/test/diff/pretyper/ucs/coverage/ConflictedPatterns.mls

```diff
@@ -0,0 +1,121 @@
+:NewDefs
+
+// This test file contains the cases where inner cases are conflicting with outer cases.
+
+class X
+class Y
+class Z
+//│ class X {
+//│   constructor()
+//│ }
+//│ class Y {
+//│   constructor()
+//│ }
+//│ class Z {
+//│   constructor()
+//│ }
+
+:w
+:ducs:normalize.result
+fun f(x) = if
+  x is X and x is Y then 1
+  else 2
+//│ Normalized UCS term:
+//│ case x*‡ of
+//│   X*◊ -> 2
+//│   _ -> 2
+//│ ╔══[WARNING] possibly conflicted patterns
+//│ ║  l.21: 	  x is X and x is Y then 1
+//│ ║        	                  ^
+//│ ╟── the scrutinee was matched against X
+//│ ║  l.21: 	  x is X and x is Y then 1
+//│ ║        	       ^
+//│ ╟── which is unrelated with Y
+//│ ║  l.21: 	  x is X and x is Y then 1
+//│ ╙──      	                  ^
+//│ fun f: Object -> 2
+
+
+:w
+:ducs:normalize.result
+fun f(x) = if
+  x is X and
+    x is Y then 1
+    x is Y then 2
+  else 3
+//│ Normalized UCS term:
+//│ case x*‡ of
+//│   X*◊ -> 3
+//│   _ -> 3
+//│ ╔══[WARNING] possibly conflicted patterns
+//│ ║  l.43: 	    x is Y then 1
+//│ ║        	         ^
```

Comment:
This location is repeated, which is a bit weird/confusing. It would be more helpful to show something like:
```suggestion
//│ ╔══[WARNING] possibly conflicted patterns for scrutinee `x`
//│ ║  l.43: 	    x is Y then 1
//│ ║        	    ^
```